cypress/e2e/login.test.ts

```diff
@@ -1,13 +1,36 @@
-describe("Navigation Test", () => {
+describe("Homepage test", () => {
 	beforeEach(() => {
 		// TODO: Add database seeding for e2e test user instead of using local profiles.
 		cy.login();
 		// Visit a route in order to allow cypress to actually set the cookie
 		cy.visit("/");
 	})
 
-	it("can direct to coachees", () => {
-		cy.visit("/coachees");
-		cy.findAllByText("资深导师职责").should("be.visible");
-	});
+	it("can render home page", () => {
+		cy.findAllByText("我的会议").should("be.visible");
+	})
+
+	it("can join meeting", () => {
+		cy.get('.css-1r08f2c').each(($ele) => {
+			cy.wrap($ele).find('button').contains("加入").click();
```

Comment:
how does it verify the web page returns a tencent meeting link after the button is clicked? Note that we recently changed the button's logic. It now opens a new browser window to open the tencent meeting link as opposed to opening it in the current browser window.

Reply:
used cy.origin to allow the domain change (by default each test should stay on the same origin). and it can verify the domain has been change to tecent.meeting.